server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Tablet.java

```diff
@@ -1555,24 +1555,24 @@ public long totalIngestBytes() {
   }
 
   public long totalQueryResultsBytes() {
-    return this.queryResultBytes.get();
+    return this.getScanMetrics().getQueryByteCount();
   }
 
   public long totalScannedCount() {
-    return this.scannedCount.get();
+    return this.getScanMetrics().getScannedCount();
   }
 
   public long totalLookupCount() {
-    return this.lookupCount.get();
+    return this.getScanMetrics().getLookupCount();
   }
 
   // synchronized?
   public void updateRates(long now) {
-    queryRate.update(now, queryResultCount.get());
-    queryByteRate.update(now, queryResultBytes.get());
+    queryRate.update(now, this.getScanMetrics().getQueryResultCount());
```

Comment:
it seems this rate will now be incorrect because its the counts of all tablets instead of the count for this tabler.